docs/variables/twitchreward.md

```diff
@@ -4,6 +4,12 @@ id: twitchreward
 
 # $(twitchreward)
 
+:::note
+
+Due to limitations imposed by the [**Twitch API**](https://dev.twitch.tv/docs/api/reference/#get-custom-reward-redemption), Fossabot is only able to return metadata for rewards created through the [**Fossabot dashboard**](https://fossabot.com/login). Create them through Fossabot's `Channel Points` page.
```

Comment:
I didn't want to paste this on every single item, it felt a little much as I started scrolling through. I think the error message is clear enough anyways.